SPECS/python-certifi/python-certifi.spec

```diff
@@ -1,13 +1,14 @@
 Summary:        Python package for providing Mozilla's CA Bundle
 Name:           python-certifi
 Version:        2023.05.07
-Release:        1%{?dist}
+Release:        2%{?dist}
 License:        MPL-2.0
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 URL:            https://certifi.io/
 Source:         https://github.com/certifi/%{name}/archive/%{version}/%{name}-%{version}.tar.gz
 Patch0:         certifi-2022.12.07-use-system-cert.patch
+Patch1:         CVE-2024-39689.patch
```

Comment:
We don't include these certificates in our package - see `rm -rf certifi/*.pem` in the `%prep` stage. This means that no change is needed, so you can dispute the CVE and closed the PR.